drizzle/schema.ts

```diff
@@ -77,3 +78,22 @@ export const participants = pgTable("participants", {
   identifier: text("identifier").notNull().unique(),
   label: text("label"),
 });
+
+export const user = pgTable("user", {
+  id: text("id").primaryKey(),
+  username: text("username").notNull().unique(),
+  passwordHash: text("password_hash").notNull(),
+});
+
+export const session = pgTable("session", {
+  id: text("id").primaryKey(),
+  userId: text("user_id")
+    .notNull()
+    .references(() => user.id),
+  expiresAt: timestamp("expires_at", {
```

Comment:
Do you think the other `date` fields (on protocols, etc) should also use `timestamp`?

Reply:
This is a complicated question - date doesn't store any time information. Most of the time we should use timestamp if there's any chance that time information will also be important. I don't think there are any scenarios where we only care about it down to the day level of resolution.

Reply:
I used `timestamp` because it was used in the Lucia docs.